concepts.tex

```diff
@@ -527,14 +533,42 @@
 \ednote{Remove table [equalitycomparable] in [utility.arg.requirements]. Replace references to
 [equalitycomparable] with [concepts.lib.compare.equalitycomparable].}
 
+{\color{newclr}
+\indexlibrary{\idxcode{BikeshedEqualityComparable}}%
+\begin{itemdecl}
+template <class T, class U>
+concept bool BikeshedEqualityComparable() {
+  return requires(const T t, const U u) {
+    { t == u } -> Boolean;
+    { u == t } -> Boolean;
+    { t != u } -> Boolean;
+    { u != t } -> Boolean;
+  };
+}
+\end{itemdecl}
+
+\begin{itemdescr}
+Let \tcode{t} and \tcode{u} be objects of types \tcode{T} and \tcode{U}.
```

Comment:
Virtual Stephan says, "What is 'object' here? What about references? Why not use `declval`?"